ocs_ci/ocs/ui/acm_ui.py

```diff
@@ -181,6 +182,49 @@ def navigate_data_services(self):
             log.error("Couldn't navigate to data Services page on ACM UI")
             raise NoSuchElementException
 
+    def navigate_disaster_recovery(self):
+        """
+        Navigate to Disaster recovery page on ACM UI
+
+        """
+        log.info("Navigate to Disaster recovery page on ACM console")
+        find_element = self.wait_until_expected_text_is_found(
+            locator=self.acm_page_nav["data-services"],
+            expected_text="Data Services",
+            timeout=240,
+        )
+        if find_element:
+            self.do_click(
+                locator=self.acm_page_nav["data-services"],
+                timeout=160,
+                avoid_stale=True,
+            )
+            disaster_recovery = self.wait_until_expected_text_is_found(
+                locator=self.acm_page_nav["disaster-recovery"],
+                expected_text="Disaster recovery",
+                timeout=240,
+            )
+            if disaster_recovery:
```

Comment:
Same as above

Reply:
If I didn't miss anything ,the [`wait_until_expected_text_is_found`](https://github.com/Shrivaibavi/ocs-ci/blob/c28600d480ef4214a487be57684716ed7ccd6f9d/ocs_ci/ocs/ui/base_ui.py#L595-L627) method didn't raise an exception but returns `False`, which means that it will silently skip the rest of this method without any notification (not sure, if that is ok, or not)

Reply:
That should be ok, if disaster_recovery = wait_until_expected_text_is_found returns false it will not proceed further

Reply:
I don't think so, as Daniel mentioned, it just returns False and any check or assertion should be added to handle the else part of it.

Reply:
added else part everywhere, marking the conversation as resolved